lib/revision-selector/index.jsx

```diff
@@ -126,8 +144,19 @@ export class RevisionSelector extends Component {
   }
 }
 
-RevisionSelector.propTypes = {
-  revisions: PropTypes.array.isRequired,
-};
-
-export default onClickOutside(RevisionSelector);
+const { setRevision, setIsViewingRevisions } = appState.actionCreators;
+
+const mapDispatchToProps = dispatch => ({
+  setRevision: revision => dispatch(setRevision({ revision })),
+  resetIsViewingRevisions: () => {
+    dispatch(setIsViewingRevisions({ isViewingRevisions: false }));
```

Comment:
Bug: When I click `Restore Note` it doesn't actually restore the revision. It should restore the note and dismiss the revisions selector.